### ISSUE TITLE
Remove or create better test that kubectl can run

### ISSUE THREAD

**Issue by rrmckinley** [kismatic/kubernetes-distro-packages#3]
The ```master.sh``` script currently produces the error below when trying to run the kubectl command. kubectl always tries to connect to the cluster. We should remove running the command or try to test what we want to know another way.

```
kubectl version
Client Version: version.Info{Major:"0", Minor:"11+", GitVersion:"v0.11.0-dirty", GitCommit:"18ddff0eb6bc069b41832ce47d19b5b89cfb26e3", GitTreeState:"dirty"}
Couldn't read version from server: Get http://localhost:8080/version: dial tcp 127.0.0.1:8080: connection refused
```

**Comment by preillyme** [comment]
@rrmckinley what do you think makes sense here as far as a test goes?

**Comment by bcbroussard** [comment]
Resolved with PR #9
